globus_automate_client/cli/helpers.py

```diff
@@ -9,7 +10,11 @@
 )
 
 
-def format_and_echo(result: Union[GlobusHTTPResponse, str], verbose=False) -> None:
+def format_and_echo(
+    result: Union[GlobusHTTPResponse, str],
+    dumper: Callable = lambda result, **args: json.dumps(result, indent=4, sort_keys=True),
```

Comment:
I liked the callable as the parameter when I read it above, this is a particularly cute approach with the lambda. Makes me wonder in other places where `json.dumps` is specifically provided if it should be removed to get this behavior, or if this lambda, and perhaps a corresponding one with indent etc. set for yaml should be defined more globally and re-used on all the calls to this function.

I guess if such lambdas or other function definitions are defined they could be in this same source file and imported into the caller.